src/settings/Settings.tsx

```diff
@@ -858,6 +858,11 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         default: true,
         controller: new IncompatibleController("showCommunitiesInsteadOfSpaces", null),
     },
+    "Spaces.includeSubSpaceRoomsInRoomList": {
+        displayName: _td("Include all sub-space rooms in Space room list"),
```

Comment:
We consider sub-space/subspace jargon and do not use it in the UI

Reply:
What language should be used? is there a guide for this somewhere?

Reply:
There is no guide given that Subspaces are still in Beta.

![image](https://user-images.githubusercontent.com/2403652/152170091-6946783a-e03c-4897-b63f-2e5f74582f81.png)

There is no existing language here but all instances of "sub-space" were previously stripped from the app so should not be re-introduced without being done holistically and intentionally.